cypress/integration/admin/pageBuilder/blockCategories/searchAndSortBlockCategories.spec.js

```diff
@@ -0,0 +1,143 @@
+import uniqid from "uniqid";
+
+const sort = {
+    NEWEST_TO_OLDEST: "createdOn_DESC",
+    OLDEST_TO_NEWEST: "createdOn_ASC",
+    A_TO_Z: "name_ASC",
+    Z_TO_A: "name_DESC"
+};
+
+context("Block Categories Module", () => {
+    const entries = [];
+
+    before(() => {
+        // Create three block categories
+        for (let i = 0; i < 3; i++) {
+            const id = uniqid();
+            cy.pbCreateBlockCategory({
+                data: {
+                    name: `${i} Cool Block Category ${id}`,
+                    slug: `block-category-slug-${id}`,
+                    icon: `block-category-icon-${id}`,
+                    description: `${i} Cool Block Category Description ${id}`
+                }
+            }).then(menu => {
+                entries.push(menu);
+            });
+        }
+    });
+
+    after(() => {
```

Comment:
I have a question related this `after` callback.

What happens if a test fail? `after` won't get called, right?

I'm asking because we have retry mechanism set up, so if this tests start running again, and there are leftover items from the previous run (b/c it failed and `after` was not called), then they will prolly fail.

What we've seen before that works is having "delete all test records" in the `before` block, and then the 'create three block categories' that you have here.

That way we ensure test data is deleted before this is run. There is less chance (if any) for leftover test records to exist.

Wdyt?